action/recommendation/index_picker.py

```diff
@@ -133,22 +134,21 @@ def write_action_batch(batch):
                             # The recommended action is not an improvement.
                             # Start a new batch.
                             break
-                        else:
-                            # The recommended action is an improvement.
-                            # Apply the recommended action.
-                            assert action not in actions_taken
-                            cursor.execute(action)
-                            conn.commit()
-                            # Log the action taken.
-                            actions_taken.append(action)
-                            print(
-                                f"{datetime.datetime.now()} "
-                                f"Applied action due to improved returns "
-                                f"(previous {previous_returns} -> "
-                                f"current {current_returns}): {action}",
-                                flush=True,
-                            )
-                            previous_returns = current_returns
+                        # The recommended action is an improvement.
```

Comment:
For documentation in PR history: these changes were part of fixing up `pylint` checks.

I accidentally pushed them in here, but we would have eventually wanted to do them anyway.